models/Project.js

```diff
@@ -162,18 +162,17 @@ Project.add({
         type: Types.Relationship,
         ref: 'Filter',
         filters: {
-            type: 'Person'
+            category: 'Person'
```

Comment:
Change to "Person(s)"

Reply:
Not sure what this means - the 'category' field on Filters is a select with one of the options being "Person" so this is filtering by all "Person" Filters